devel/icu-devel/Portfile

```diff
@@ -116,10 +116,20 @@ if {${subport} eq ${name} || ${subport} eq "${name}-lx"} {
     test.target     check
     test.args       VERBOSE=1
 
+    # icu4c requires python2.7+ regardless of configure:
+    #   checking for python3...
+    # libxml which is used to compile llvm/clang depends on icu,
```

Comment:
It's libxml2, not libxml. The libxml port is for the ancient 1.x version.

Reply:
thanks for catching such typo